src/packet/bwe/rate_control.rs

```diff
@@ -34,18 +37,18 @@ struct RateControl {
     /// The averaged observed bitrate when we have been in the decrease state.
     averaged_observed_bitrate: MovingAverage,
     /// The last time we updated the estimated bitrate.
-    last_estimate_update: Instant,
+    last_estimate_update: Option<Instant>,
     // Last RTT estimate in micro-seconds
-    last_rtt_us: Option<f64>,
+    last_rtt: Option<Duration>,
+
+    is_probing: bool,
+    stoppped_probing_at: Option<Instant>,
```

Comment:
The need for this needs to be revisited, it was a bit of a hack when I wasn't correctly generating padding near the estimated rate. Shouldn't be required anymore